db/migrate/20201008204114_add_required_tags_to_katello_root_repositories.rb

```diff
@@ -0,0 +1,5 @@
+class AddRequiredTagsToKatelloRootRepositories < ActiveRecord::Migration[6.0]
+  def change
+    add_column :katello_root_repositories, :required_tags, :string
```

Comment:
```suggestion
    add_column :katello_root_repositories, :required_tags, :text
```

https://medium.com/@songyishu.yang/increase-text-limit-in-rails-44dc254f7c0b